types/events.go

```diff
@@ -21,6 +24,11 @@ type Stats struct {
 	NetworkInterfaces []*NetworkInterface `json:"network_interfaces"`
 }
 
+type (
+	PSIData  = cgroups.PSIData
```

Comment:
FWIW, not sure if we need the aliases at all; I need to look into history to understand why we did so for the other ones (separate types from libcontainer); perhaps we don't need those either.

It _may_ have been to allow external consumer to use the "types" package without requiring libcontainer, but if that was the design, we already broke that.

If, on the other hand, nobody is expected to consume "types", we may not need all the duplicated definitions and we could just alias or use them directly

Reply:
We expect `types` to be consistent i.e. not depending on `libcontainer` internals, i..e consumer of `runc events` exect the format to be consistent even though we may have change `libcontainer/cgroups` for some reasons.

So some of the duplication is intentional, and some differences are also intentional i.e. `camelCase` vs `snake_case`.